nexus_keycode/protocols/full.py

```diff
@@ -363,6 +382,41 @@ def display_payg_id(cls):
         Message contians no body.
 
         :return: Message object of format FACTORY_DISPLAY_PAYG_ID_TEST
-        :rtype: :class:`FactoryMessage`
+        :rtype: :class:`FactoryFullMessage`
         """
         return cls(message_type=FullMessageType.FACTORY_DISPLAY_PAYG_ID, body="")
+
+    @classmethod
+    def passthrough_command(cls, application_id, passthrough_digits):
+        # type: (PassthroughApplicationId, str)-> FactoryFullMessage
+        """Send a keycode which contains application-specific data, and
+        will not be parsed by the embedded keycode library. Passthrough
+        commands do not trigger any UI feedback (keycode accepted/etc) from the
+        Nexus Keycode firmware library, and instead defer any activity at all to
+        the final application which receives and parses the passthrough
+        command.
+
+        Warning: passthrough commands *do not* have any MAC, and are not
+        validated in any way by the Nexus library in devices - the passthrough
+        `subtype ID` is examined, and the message is forwarded onward
+        accordingly. Applications that use passthrough command should include
+        integrity checks on the transmitted data inside the message body.
+
+        :param application_id: ID of device application processing this command
+        :type application_id: :class:`PassthroughApplicationId`
+        :type passthrough_digits: :class:`string`
+        :return: Message object of format PASSTHROUGH_COMMAND
+        :rtype: :class:`FactoryFullMessage`
+        """
+        if not isinstance(application_id, PassthroughApplicationId):
+            raise TypeError("Passthrough command requires an application ID.")
```

Comment:
We could also add a unit test here to check this. For example, try to create a passthrough command with the `application_id` argument equal to 2.